images/Dockerfile.openscope

```diff
@@ -0,0 +1,75 @@
+FROM quay.io/jupyter/datascience-notebook:python-3.11.10
+
+USER root
+
+RUN apt update \
+ && apt install -y software-properties-common \
+ && add-apt-repository -y 'ppa:apptainer/ppa' \
+ && apt update \
+ && apt install -y apptainer-suid \
+ && rm -rf /var/lib/apt/lists/* && rm -rf /tmp/*
+
+RUN apt-get update && apt-get install -y ca-certificates libseccomp2 \
+   s3fs netbase less parallel tmux screen vim emacs htop curl \
+   && rm -rf /var/lib/apt/lists/* && rm -rf /tmp/*
+
+RUN curl --silent --show-error "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" \
+  -o "awscliv2.zip" && unzip awscliv2.zip \
+  && ./aws/install && rm -rf ./aws awscliv2.zip
+
+# Install jupyter server proxy and desktop
+RUN curl -fsSLo /usr/share/keyrings/brave-browser-archive-keyring.gpg https://brave-browser-apt-release.s3.brave.com/brave-browser-archive-keyring.gpg \
+   && echo "deb [signed-by=/usr/share/keyrings/brave-browser-archive-keyring.gpg] https://brave-browser-apt-release.s3.brave.com/ stable main"|sudo tee /etc/apt/sources.list.d/brave-browser-release.list \
+   && apt-get -y -qq update \
+   && apt-get -y -qq install \
+        dbus-x11 \
+        xfce4 \
+        xfce4-panel \
+        xfce4-session \
+        xfce4-settings \
+        xorg \
+        xubuntu-icon-theme \
+        brave-browser \
+        tigervnc-standalone-server \
+        tigervnc-xorg-extension \
+    # chown $HOME to workaround that the xorg installation creates a
+    # /home/jovyan/.cache directory owned by root
+   && chown -R $NB_UID:$NB_GID $HOME \
+   && rm -rf /var/lib/apt/lists/* && rm -rf /tmp/*
+
+# apt-get may result in root-owned directories/files under $HOME
+RUN mkdir /opt/extras && chown -R $NB_UID:$NB_GID $HOME /opt/extras
+
+USER $NB_USER
+
+RUN pip install --no-cache-dir jupyter-remote-desktop-proxy
+
+ENV MAMBA_NO_LOW_SPEED_LIMIT=1
+
+RUN mamba install --yes 'datalad>=0.18' rclone 'h5py>3.3=mpi*' ipykernel zarr blosc eccodes websockify \
+  && wget --quiet https://raw.githubusercontent.com/DanielDent/git-annex-remote-rclone/v0.8/git-annex-remote-rclone \
+  && chmod +x git-annex-remote-rclone && mv git-annex-remote-rclone /opt/conda/bin \
+  && conda clean --all -f -y && rm -rf /tmp/*
+
+RUN pip install --no-cache-dir -U plotly jupyter_bokeh jupytext nbgitpuller datalad_container \
+    datalad-osf dandi nibabel nilearn pybids spikeinterface neo itkwidgets \
+    'pydra>=0.25' 'pynwb>=2.8.3' 'nwbwidgets>=0.10.2' hdf5plugin s3fs h5netcdf "xarray[io]"  \
+    aicsimageio kerchunk 'neuroglancer>=2.28' cloud-volume ipywidgets ome-zarr \
+    webio_jupyter_extension https://github.com/balbasty/dandi-io/archive/refs/heads/main.zip \
+    tensorstore anndata
+
+RUN pip install --no-cache-dir multiscale_spatial_image jupyterlab-git \
+        datalad-fuse
+
+RUN pip install --no-cache-dir brainrender napari
+
+# Leaving "standard" Dockerfile copy above as-is to simplify later switch to modular builds.
```

Comment:
```suggestion
FROM dandiarchive/dandihub:latest
```

Reply:
@kabilar this is precisely what I want to do, but I don't think its "safe" to do so yet. IIUC, these jobs all run in parallel, and the change you proposed could pull the `latest` from Dockerhub rather than `latest` that is built here. 

So we need a way to ensure that the "Standard" aka `dandihub:latest` is built fresh prior to being used by any subsequent Dockerfiles. IIUC the other images could be significantly simplified to do this, so it would probably be best to do that work all at once in a separate PR.

Reply:
Ah yes thats right.  Please ignore.  Thanks.

Reply:
opened https://github.com/dandi/dandi-hub/issues/235